docs/_providers/dnsme.md

```diff
@@ -0,0 +1,62 @@
+---
+name: DNSME
+title: DNS Made Simple Provider
+layout: default
+jsId: DNSME
+---
+# DNS Made Simple Provider
+
+## Configuration
+In your credentials file, you must provide your `api_key` and `secret_key`. More info about authentication can be found in [DNS Made Easy API docs](https://api-docs.dnsmadeeasy.com/).
+
+{% highlight json %}
+{
+  "dnsme": {
+    "api_key": "1c1a3c91-4770-4ce7-96f4-54c0eb0e457a",
+    "secret_key": "e2268cde-2ccd-4668-a518-8aa8757a65a0"
+  }
+}
+{% endhighlight %}
+
+## Records
+
+Changing apex NS records is not supported.
+
+ALIAS/ANAME records are supported.
+
+This provider does not support HTTPRED records.
+
+SPF records are ignored by this provider. Use TXT records instead.
+
+## Metadata
+This provider does not recognize any special metadata fields unique to DNS Made Easy.
+
+## Usage
+Example Javascript:
+
+{% highlight js %}
+var REG_NONE = NewRegistrar('none', 'NONE')
+var DNSME = NewDnsProvider("dnsme", "DNSME");
+
+D("example.tld", REG_NONE, DnsProvider(DNSME),
+    A("test","1.2.3.4")
+);
+{%endhighlight%}
+
+## Activation
+You can generate you `api_key` and `secret_key` in [Control Panel](https://cp.dnsmadeeasy.com/) in Account Information in Config menu.
+
+API is only available for Business plan and higher plans.
+
+## Caveats
+
+### Global Traffic Director
+Global Traffic Director feature is not supported.
+
+## Development
+
+### Debugging
+Set `DNSME_DEBUG` environment variable to dump all API calls made by this provider.
+
+### Testing
+Set `sandbox` key to `"true"` (this must be string, not a boolean) in credentials JSON alongside `api_key` and `secret_key` to make all API calls against DNS Made Easy sandbox environment.
```

Comment:
This seems like an easy mistake for humans to make.  How about if `sandbox` is non-null?

Reply:
Oh, my bad! There actually is a condition to compare `settings["sandbox"] != ""` instead of `"true"` string now. I forgot to update the docs when I changed it, sorry.